src/desc.html

```diff
@@ -1,16 +1,5 @@
+<b>Ultimate Geography v4 (LEGACY)</b>
 
-<a href="https://github.com/axelboc/anki-ultimate-geography/"><b>FULL DESCRIPTION</b></a> | <a href="https://github.com/axelboc/anki-ultimate-geography/releases"><b>RELEASE NOTES</b></a> | <a href="https://github.com/axelboc/anki-ultimate-geography/blob/master/CONTRIBUTING.md"><b>CONTRIBUTING</b></a>
+This deck holds content that was removed as part of Ultimate Geography's v4 release. If you're not interested in this content, you may delete this deck.
```

Comment:
Should there perhaps be slightly more detail here? 

In principle, if somebody imports the legacy deck, then it means that they'll have seen the release notes, and hopefully read them, but if they did things in a hurry, they might possibly get confused (e.g. why are there "new" cards?). Obviously, if they get confused they should click on the link, but I know that I myself can behave irrationally when panicking...

Perhaps:

```suggestion
This deck holds content that was removed from the main deck as part of Ultimate Geography's v4 release. It will no longer be updated by the project. If you're not interested in it, you may delete this deck.

Some of your progress for the cards in the removed content has had to be reset, which is why some cards are marked as "new". (No progress will have been lost in the main deck.)
```

(I'd add "has had to be reset for technical reasons", but that sounds awfully weaselly...)

Obviously, this is just a suggestion!

Reply:
Sounds much clearer, thanks 👍